src/plugins/Makefile.am

```diff
@@ -121,6 +125,14 @@ libgs_plugin_dpkg_la_LIBADD = $(GS_PLUGIN_LIBS)
 libgs_plugin_dpkg_la_LDFLAGS = -module -avoid-version
 libgs_plugin_dpkg_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(WARN_CFLAGS)
 
+if HAVE_EXTERNAL_APPSTREAM
```

Comment:
You don't need this conditional. Only the above conditional that adds it to `plugin_LTLIBRARIES` is needed. The rules defined here don't need to be commented out.